livechat/utils/helpers.py

```diff
@@ -16,7 +16,7 @@ def prepare_payload(parameters: dict) -> dict:
     '''
     return {
         key: value
-        for key, value in parameters.items()
-        if key not in ['self', 'payload', 'headers', 'date_to', 'date_from']
+        for key, value in parameters.items() if key not in
+        ['self', 'payload', 'token', 'headers', 'date_to', 'date_from']
```

Comment:
it requires adding:
```
        if token:
            payload['token'] = str(token)
```
several times. what is the reason here? do you think the current way of adding a token to the payload is too implicit or is it a matter of type conversion to `str`?